nbs/methods.ipynb

```diff
@@ -44,6 +44,7 @@
     "\n",
```

Comment:
<p>See previous comment on the <code>sparsity</code> attribute. If it works, it should be included in all the methods.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>